core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -81,6 +84,19 @@ object KafkaServer {
     clientConfig
   }
 
+  def zkClient(name: String, time: Time, config: KafkaConfig, zkClientConfig: ZKClientConfig): KafkaZkClient = {
```

Comment:
This is coming from #12946 which is getting merged first

Reply:
Hmm. Maybe I'm missing something, but I don't see any relationship with `KafkaServer`. Why put this in `KafkaServer.scala`? Can this be a `static` function in `KafkaZkClient.scala`? (aka "function on `object KafkaZkClient`")?

Reply:
This was just relocated from elsewhere in KafkaServer. I do agree it's kind of out of place. I'll move it.